index.js

```diff
@@ -119,16 +119,17 @@ module.exports = (opts = {}) => {
 	});
 };
 
-module.exports.download = (win, url, opts) => new Promise((resolve, reject) => {
-	opts = Object.assign({}, opts, {unregisterWhenDone: true});
+module.exports.download = (win, url, opts) =>
+	new Promise((resolve, reject) => {
+		opts = Object.assign({}, opts, { unregisterWhenDone: true });
 
-	registerListener(win.webContents.session, opts, (err, item) => {
-		if (err) {
-			reject(err);
-		} else {
-			resolve(item);
-		}
-	});
+		registerListener(win.webContents.session, opts, (err, item) => {
```

Comment:
Don't do unrelated changes.

Reply:
let me start fresh, i don't know why this happened.